requirements.txt

```diff
@@ -1,2 +1,3 @@
 numpy==1.19.4
 cython==0.29.21
+parameterized==0.8.1
```

Comment:
This is already in `tests/requirements.txt`

Reply:
Thanks, fixed.